tests/YearWeekTest.php

```diff
@@ -395,4 +399,31 @@ public function testToString(int $year, int $week, string $expected)
         $yearWeek = YearWeek::of($year, $week);
         $this->assertSame($expected, (string) $yearWeek);
     }
+
+    public function testNow()
+    {
+        $now = new FixedClock(Instant::of(2000000000));
+        $timeZone = TimeZone::parse('Asia/Taipei');
+        $yearWeek = YearWeek::now($timeZone, $now);
+
+        $this->assertSame('2033-W20', (string) $yearWeek);
+    }
+
+    public function testGetFirstDay()
+    {
+        $yearWeek = YearWeek::of(2015, 1);
+        $result = $yearWeek->getFirstDay();
+
+        $this->assertInstanceOf(LocalDate::class, $result);
```

Comment:
This line can be removed, `assertLocalDate()` is typed and will throw a TypeError if it is not a LocalDate.